src/components/Schedule/Schedule.css

```diff
@@ -35,13 +36,58 @@
 	font-weight: normal;
 }
 
+.schedule-btn {
+	background: var(--button-gradient);
+	border-radius: 100px;
+	color: var(--medium-primary-color);
+	text-transform: none;
+	min-width: 16%;
+	transition: 1s;
+	cursor: pointer;
+	border: 0;
+}
+
+.schedule-btn:hover {
+	background-color: #f094bc;
+}
+
+.schedule-btn:disabled {
+	cursor: default;
+	color: var(--disabled-text-btn-color);
+	background: var(--button-gradient);
+	transition: none;
+}
+
 .button-text {
 	font-family: Rubik;
 	margin: auto;
 	font-weight: bold;
 	font-size: 15px;
 }
 
+.schedule-card {
+	border: 1px solid black;
+	transition: transform 330ms ease-in-out;
+	box-shadow: rgba(0, 0, 0, 0.15) 0px 15px 25px, rgba(0, 0, 0, 0.05) 0px 5px 10px;
+	border: 3px solid white;
+	border-radius: 10px;
+	max-height: 140px;
+	min-width: 150px;
+	max-width: 150px;
```

Comment:
can these be measured with non-fixed units?
perhaps remove `max` and `min` to use regular widths and heights?